config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "homepage#index", as: 'home'
+
+  resources :works, :votes
+
+  resources :users, only: [:index, :show]
+
+  post '/works/:work_id/votes', to: 'votes#create', as: 'create_work_vote'
```

Comment:
Nice! This is truly the only votes route we need. Overall, these routes look like exactly what we wanted